video/types.h

```diff
@@ -299,3 +299,19 @@ uint32_t convertFloatToValue(float rawValue, bool is16Bit, bool isFixed, int8_t
 	}
 };
 
+// This returns an INT32 because it will return either the value
+// in the range 0-65535 or -1 if the string is invalid.
+int32_t textToInt32(const char * text) {
+    int32_t val = 0;
+    int8_t digit = 0;
+
+	if (!isdigit(text[0]))
+		{ debug_log("convert to ASCII text %s invalid\n\r", text); return -1; };
+	while (isdigit(text[digit]) && digit<6)
+	{
+		val = (val * 10) + text[digit] - '0';digit++;
+	};
+	debug_log("contverted text %s is %u\n\r", text,val);
```

Comment:
bike-shed comments would be that the codebase generally has the open squiggles on the same line as the `if` and `while` statements, with a new line immediately after... plus the increment of `digit` should be on a new line...  and that the final `debug_log` message here has a typo 😀  (should be "converted")

Reply:
Makes sense - I'll keep the formatting consistent per this advice.